samples/algorithms/integer-factorization/QuantumVizEstimator.cs

```diff
@@ -12,10 +12,10 @@
 
 namespace Microsoft.Quantum.Simulation.Simulators
 {
-    public class QuantumVizEstimator : ResourcesEstimator
+    public class QuantumVizEstimator : QCTraceSimulator
```

Comment:
Same thing here, perhaps this would be better removed if the tracer behavior is too different?

Reply:
In this particular instance no additional functionality of ResourcesEstimator is used. QCTraceSimulator works just as well. But see other notes.